### ISSUE TITLE
Centroid model is not generating metrics files

### ISSUE THREAD

**Issue by mittner7** [murthylab/sleap#599]
Hello,
I'm trying to run analysis on my models, but the centroid model does not contain metrics files. 
I'm running version 1.1.5 and using Colab for training and predicting. My centered_instance file does contain the metrics, and when running previous versions I saw them in centroid as well.

`models/centroid2
├── best_model.h5
├── initial_config.json
├── labels_gt.train.slp
├── labels_gt.val.slp
├── train
│   ├── events.out.tfevents.1634938180.91c71cefa6b6.1995.3493.v2
│   └── events.out.tfevents.1634938185.91c71cefa6b6.1995.7280.v2
├── training_config.json
├── training_log.csv
└── validation
    ├── events.out.tfevents.1634938180.91c71cefa6b6.1995.3501.v2
    └── events.out.tfevents.1634938251.91c71cefa6b6.1995.15839.v2

2 directories, 10 files`

`{
    "data": {
        "labels": {
            "training_labels": null,
            "validation_labels": null,
            "validation_fraction": 0.1,
            "test_labels": null,
            "search_path_hints": [],
            "skeletons": []
        },
        "preprocessing": {
            "ensure_rgb": false,
            "ensure_grayscale": false,
            "imagenet_mode": null,
            "input_scaling": 0.5,
            "pad_to_stride": null
        },
        "instance_cropping": {
            "center_on_part": null,
            "crop_size": 160,
            "crop_size_detection_padding": 16
        }
    },
    "model": {
        "backbone": {
            "leap": null,
            "unet": {
                "stem_stride": null,
                "max_stride": 16,
                "output_stride": 2,
                "filters": 16,
                "filters_rate": 2.0,
                "middle_block": true,
                "up_interpolate": true,
                "stacks": 1
            },
            "hourglass": null,
            "resnet": null
        },
        "heads": {
            "single_instance": null,
            "centroid": {
                "anchor_part": "",
                "sigma": 5.0,
                "output_stride": 2
            },
            "centered_instance": null,
            "multi_instance": null
        }
    },
    "optimization": {
        "preload_data": true,
        "augmentation_config": {
            "rotate": true,
            "rotation_min_angle": -180.0,
            "rotation_max_angle": 180.0,
            "translate": false,
            "translate_min": -5,
            "translate_max": 5,
            "scale": false,
            "scale_min": 0.9,
            "scale_max": 1.1,
            "uniform_noise": false,
            "uniform_noise_min_val": 0.0,
            "uniform_noise_max_val": 10.0,
            "gaussian_noise": false,
            "gaussian_noise_mean": 5.0,
            "gaussian_noise_stddev": 1.0,
            "contrast": false,
            "contrast_min_gamma": 0.5,
            "contrast_max_gamma": 2.0,
            "brightness": false,
            "brightness_min_val": 0.0,
            "brightness_max_val": 10.0
        },
        "online_shuffling": true,
        "shuffle_buffer_size": 128,
        "prefetch": true,
        "batch_size": 4,
        "batches_per_epoch": null,
        "min_batches_per_epoch": 200,
        "val_batches_per_epoch": null,
        "min_val_batches_per_epoch": 10,
        "epochs": 200,
        "optimizer": "adam",
        "initial_learning_rate": 0.0001,
        "learning_rate_schedule": {
            "reduce_on_plateau": true,
            "reduction_factor": 0.5,
            "plateau_min_delta": 1e-06,
            "plateau_patience": 5,
            "plateau_cooldown": 3,
            "min_learning_rate": 1e-08
        },
        "hard_keypoint_mining": {
            "online_mining": false,
            "hard_to_easy_ratio": 2.0,
            "min_hard_keypoints": 2,
            "max_hard_keypoints": null,
            "loss_scale": 5.0
        },
        "early_stopping": {
            "stop_training_on_plateau": true,
            "plateau_min_delta": 1e-06,
            "plateau_patience": 10
        }
    },
    "outputs": {
        "save_outputs": true,
        "run_name": "211011_144622",
        "run_name_prefix": "",
        "run_name_suffix": "",
        "runs_folder": "",
        "tags": [
            ""
        ],
        "save_visualizations": true,
        "log_to_csv": true,
        "checkpointing": {
            "initial_model": false,
            "best_model": true,
            "every_epoch": false,
            "latest_model": false,
            "final_model": false
        },
        "tensorboard": {
            "write_logs": true,
            "loss_frequency": "epoch",
            "architecture_graph": false,
            "profile_graph": false,
            "visualizations": true
        },
        "zmq": {
            "subscribe_to_controller": false,
            "controller_address": "tcp://127.0.0.1:9000",
            "controller_polling_timeout": 10,
            "publish_updates": false,
            "publish_address": "tcp://127.0.0.1:9001"
        }
    }
}`

**Comment by talmo** [comment]
Update: This will be fixed in v1.2.0 and currently works in the prerelease: https://github.com/murthylab/sleap/releases/tag/v1.2.0a2

We'll update and close this issue once it's fully released.
